### PR TITLE
Extend order payload with allocations

### PR DESCRIPTION
**Pull request by d-wysocki**
I want to merge this change because...I want to extend order payload with warehouse ids and quantity allocated

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3255 Add multichannel ID and warehouse ID to order json](https://app.clickup.com/t/2549495/SALEOR-3255)

File: saleor/webhook/payloads.py [hunk 1]
```diff
@@ -90,6 +90,12 @@ def generate_order_lines_payload(lines: Iterable[OrderLine]):
         extra_dict_data={
             "total_price_net_amount": (lambda l: l.total_price.net.amount),
             "total_price_gross_amount": (lambda l: l.total_price.gross.amount),
+            "allocations": list(
+                lines.values(  # type: ignore
+                    warehouse_id=F("allocations__stock__warehouse_id"),
```
[the remark this answers is not in the document]
Comment (d-wysocki) [reply saleor/webhook/payloads.py:95]: We should use GraphQL ID for UUID field?

[the remark this answers is not in the document]
Comment (maarcingebala) [reply saleor/webhook/payloads.py:95]: Neither UUID nor regular IDs are not exposed in API. Warehouse IDs in API are underneath UUIDs.

**Review by fowczarek — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: saleor/webhook/payloads.py [hunk 2]
```diff
@@ -65,6 +65,20 @@
 )
 
 
+def prepare_order_lines_allocations_payload(lines: Iterable[OrderLine]):
+    warehosue_id_quantity_allocated_map = list(
```
Comment (maarcingebala) [inline saleor/webhook/payloads.py:69]:
```suggestion
    warehouse_id_quantity_allocated_map = list(
```

File: saleor/webhook/payloads.py [hunk 3]
```diff
@@ -65,6 +65,20 @@
...
+        lines.values(  # type: ignore
+            warehouse_id=F("allocations__stock__warehouse_id"),
+            quantity_allocated=F("allocations__quantity_allocated"),
+        )
+    )
+    for item in warehosue_id_quantity_allocated_map:
```
Comment (maarcingebala) [inline saleor/webhook/payloads.py:75]:
```suggestion
    for item in warehouse_id_quantity_allocated_map:
```

File: saleor/webhook/payloads.py [hunk 4]
```diff
@@ -65,6 +65,20 @@
...
+        item["warehouse_id"] = graphene.Node.to_global_id(
+            "Warehouse", item["warehouse_id"]
+        )
+    return warehosue_id_quantity_allocated_map
```
Comment (maarcingebala) [inline saleor/webhook/payloads.py:79]:
```suggestion
    return warehouse_id_quantity_allocated_map
```

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**